zerver/lib/test_runner.py

```diff
@@ -152,16 +153,16 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         self.failed_tests = []  # type: List[str]
 
     def addInfo(self, test: TestCase, msg: str) -> None:
-        self.stream.write(msg)
-        self.stream.flush()
+        self.stream.write(msg)  # type: ignore
+        self.stream.flush()  # type: ignore
```

Comment:
Generally, it's best to do separate commits for this sort of change as part of an upgrade, with commits explaining why it's required and the right fix?